.vscode/settings.json

```diff
@@ -38,4 +38,6 @@
 	"typescript.format.insertSpaceAfterOpeningAndBeforeClosingEmptyBraces": false,
 	"javascript.preferences.importModuleSpecifierEnding": "js",
 	"typescript.preferences.importModuleSpecifierEnding": "js",
+	"editor.tabSize": 2,
+	"editor.renderWhitespace": "boundary",
```

Comment:
let's remove these changes to the vscode settings.

we want to leave tab size and rendering options up to each user's own settings, rather than enforcing them in the project's overriding settings.

Reply:
oh... right...